### PR TITLE
Update README for 3.x

### PR DESCRIPTION
**Pull request by kevinpschaaf**
Update README for 3.x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,228 +2,127 @@
...
 
-Polymer 2.0 is released, and will be the future focus of Polymer development going forward.  We intend to keep the 2.x public API stable barring critical issues.  For background and migration information on the 2.x see the [2.0 documentation](https://www.polymer-project.org/2.0/docs/about_20) on the website or the [2.0 section below](#2-0), and we welcome your feedback via [issues](https://github.com/Polymer/polymer/issues/new) or [Slack](https://polymer-slack.herokuapp.com/).
+    * Runs in all [supported](#supported-browsers) browsers: [StackBlitz](https://stackblitz.com/edit/polymer-element-example?file=index.js), [Glitch](https://glitch.com/edit/#!/polymer-element-example?path=index.html)
 
-**To evaluate Polymer 2.0**, please point your bower at the latest `2.0.0` tag for polymer, and be sure load to the `webcomponentsjs/webcomponents-lite.js` or `webcomponentsjs/webcomponents-loader.js` polyfills from the latest `v1.0.0` tag of [`webcomponentsjs`](https://github.com/webcomponents/webcomponentsjs).
+    * Runs in browsers with [Javascript Modules](https://caniuse.com/#search=modules): [JSBin](https://jsbin.com/wuxejiz/edit?html,output),
```
Comment (keanulee) [inline README.md]: JavaScript (casing), here and everywhere

File: README.md [hunk 2]
```diff
@@ -2,228 +2,131 @@
...
-}
+## Minimal Example
 
-// Register custom element class with browser
-customElements.define('my-element', MyElement);
-```
+  1. Create a class that extends `PolymerElement`.
+  1. Implement a static `properties` getter that describes the element's public property/attribute API
+  (these automatically become observed attributes).
+  1. Then implement a `template` getter that reutrns an `HTMLTemplateElement` describing the element's rendering, including encapsulated styling and any property bindings.
```
Comment (katejeffreys) [inline README.md]: `reutrns` -> `returns`

File: README.md [hunk 3]
```diff
@@ -2,228 +2,131 @@
...
-    };
-  }
+Web components are an incredibly powerful new set of primitives baked into the web platform, and open up a whole new world of possibility when it comes to componentizing front-end code and easily creating powerful, immersive, app-like experiences on the web.
 
-}
+Polymer is a lightweight library built on top of the web standards-based [Web Components](http://webcomponents.org/introduction) APIs, and makes it easier to build your very own custom HTML elements. Creating reusable custom elements - and using elements built by others - can make building complex web applications easier and more efficient.
 
-customElements.define(MyBoundNamecard.is, MyBoundNamecard);
-```
+By being based on the Web Components API's built in the browser (or [polyfilled](https://github.com/webcomponents/webcomponentsjs) where needed), elements built with Polymer are:
```
Comment (katejeffreys) [inline README.md]: `Web Components API's` should not have possessive

File: README.md [hunk 4]
```diff
@@ -2,228 +2,131 @@
...
 
-```html
-<!-- using the element -->
-<my-bound-namecard my-name="Josh"></my-bound-namecard>
-```
+* Interoperable at the browser level
+* Built from the platform up
+* Self-contained
+* Don't require an overarching framework - are interoperable across frameworks
```
Comment (katejeffreys) [inline README.md]: `* Don't require an overarching framework - are interoperable across frameworks`

->

`* Interoperable across frameworks`

File: README.md [hunk 5]
```diff
@@ -2,228 +2,131 @@
...
 
-  static get properties() {
-    return {
-      myName: String
-    };
-  }
+## About Polymer 3.0
 
-}
+Polymer 3.0 is now released to stable, and introduces a major change to how Polymer is distributed: from HTML Imports on bower, to JS modules on npm.  Otherwise, the API is virtually 100% backward compatible with Polymer 2.0 (the only changes are removing API's related to HTML Imports like `importHref`, and converting Polymer's API to be module-based rather than globals-based).
```
Comment (katejeffreys) [inline README.md]: `from HTML Imports on bower` -> `from HTML Imports on Bower`

Comment (katejeffreys) [reply README.md]: `virtually 100%` maybe -> `almost entirely` or something? Or remove it

Comment (katejeffreys) [inline README.md]: `API's` -> `APIs`

File: README.md [hunk 6]
```diff
@@ -2,228 +2,131 @@
...
 
-customElements.define(MyStyledNamecard.is, MyStyledNamecard);
-```
-```html
-<!-- using the element -->
-<my-styled-namecard my-name="Jesse"></my-styled-namecard>
-```
+Migrating to Polyme 3.0 by hand is mostly mechanical:
```
Comment (katejeffreys) [inline README.md]: `Polyme 3.0` -> `Polymer 3.0`

File: README.md [hunk 7]
```diff
@@ -2,228 +2,131 @@
...
+* Components should be defined in JS modules instead of in HTML
+* Templates should be encoded in JS modules using a `static get template()` getter on PolymerElement subclasses using the `html` tagged template literal function (which parses `HTMLTemplateElement`'s out of strings in JS) rather than using `<template>` elements in a `<dom-module>`
```
Comment (katejeffreys) [inline README.md]: `HTMLTemplateElement`'s -> `HTMLTemplateElement`s

File: README.md [hunk 8]
```diff
@@ -2,228 +2,131 @@
...
+* All dependencies should be imported JS module imports rather than from HTML Imports.
```
Comment (katejeffreys) [inline README.md]: delete `from`

**Pull request merged**